File: 2137-FinalValueOfVariableAfterPerformingOperations/2137-FinalValueOfVariableAfterPerformingOperations.py
# Last updated: 7/21/2026, 7:08:21 PM
class Solution(object):
    def finalValueAfterOperations(self, operations):
        """
        :type operations: List[str]
        :rtype: int
        """
        return sum(1 if op[1]=='+' else -1 for op in operations)

File: 2137-FinalValueOfVariableAfterPerformingOperations/2137-FinalValueOfVariableAfterPerformingOperations.cpp
// Last updated: 7/21/2026, 7:08:24 PM
class Solution {
public:
    int finalValueAfterOperations(vector<string>& operations) {
        int x=0;
        for(auto& op: operations)
            x+=(op[1]=='+')?1:-1;
        return x;
    }
};